src/Stanford/Moss.hs

```diff
@@ -80,27 +80,44 @@ data Language
     | Modula2
     | Ada
     | Perl
-    | TCL
```

Comment:
The moss website lists TCL as a supported language. Have you removed this because you tried it and it didn't work? (I note that it and HCL2 are mentioned on the website, but not in the perl script.)

Reply:
@mbg indeed, I spotted this too.

The web page includes TCL, but the perl script (as of 24 October 2019) Stanford send out does not include it:

    @languages = ("c", "cc", "java", "ml", "pascal", "ada", "lisp", "scheme", "haskell", "fortran", "ascii", "vhdl", "perl", "matlab", "python", "mips", "prolog", "spice", "vb", "csharp", "modula2", "a8086", "javascript", "plsql", "verilog");

Should this Haskell moss library include TCL, or not?

Reply:
The comment in the perl script I received (6 February 2018) notes that the website should be the authoritative source for supported languages:
```perl
# As of the date this script was written, the following languages were supported.  This script will work with 
# languages added later however.  Check the moss website for the full list of supported languages.
```
If that's still the case in the script you received, I think it would be best to leave TCL in as I suspect that they have updated the website, but not the script whenever it was they added TCL.